public-api/src/main/java/org/pmiops/workbench/service/CdrVersionService.java

```diff
@@ -0,0 +1,31 @@
+package org.pmiops.workbench.service;
+
+import org.pmiops.workbench.cdr.CdrVersionContext;
+import org.springframework.stereotype.Service;
+import org.springframework.beans.factory.annotation.Autowired;
+import javax.inject.Provider;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.pmiops.workbench.db.model.CdrVersion;
+import org.pmiops.workbench.db.dao.CdrVersionDao;
+
+@Service
+public class CdrVersionService {
+    private CdrVersionDao cdrVersionDao;
+
+    @Autowired
+    public CdrVersionService(CdrVersionDao cdrVersionDao) {
+        this.cdrVersionDao = cdrVersionDao;
+    }
+
+    @Autowired
+    @Qualifier("defaultCdr")
+    private Provider<CdrVersion> defaultCdrVersionProvider;
+
+    public void setDefaultCdrVersion() {
+        CdrVersionContext.setCdrVersionNoCheckAuthDomain(defaultCdrVersionProvider.get());
+    }
+
+    public CdrVersion findByIsDefault(boolean isDefault) {
+        return cdrVersionDao.findByIsDefault(true);
```

Comment:
I would use the CdrVersionMapper here and return the client CdrVersion model object.

Reply:
Done thanks.